projects/kserve/visualizations/kserve-llm/plotting/errors.py

```diff
@@ -169,7 +182,10 @@ def generateFinishReasonData(entries, variables):
             #elif reason == 1: # MAX_TOKENS: good, ignore
             #    continue
             else:
-                reason = STOP_REASONS[reason]
+                if isinstance(reason, str): # vLLM
+                    reason = STOP_REASONS[convertvLLMFinishReason(reason)]
+                else:
+                    reason = STOP_REASONS[reason]
```

Comment:
I think you shouldn't go `vllm->tgis->reason`, but rather have
* `STOP_REASONS_TGIS`
* `STOP_REASONS_VLLM`